src/data/sdkReferences/posthog-js-references.json

```diff
@@ -0,0 +1,4098 @@
+                    "returnType": {
+                        "id": "void",
+                        "name": "void"
+                    },
+                    "path": "lib/src/posthog-core.d.ts"
+                },
+                {
+                    "category": "",
+                    "details": null,
+                    "id": "reloadFeatureFlags",
+                    "showDocs": true,
+                    "title": "reloadFeatureFlags",
+                    "examples": [
+                        {
+                            "id": "reloadfeatureflags",
+                            "name": "Generated example for reloadFeatureFlags",
+                            "code": "// Generated example for reloadFeatureFlags\nposthog.reloadFeatureFlags();"
+                        }
+                    ],
+                    "releaseTag": "public",
+                    "params": [],
+                    "returnType": {
+                        "id": "void",
+                        "name": "void"
+                    },
+                    "path": "lib/src/posthog-core.d.ts"
+                },
+                {
+                    "category": "",
+                    "description": "Render a survey on a specific element.",
+                    "details": null,
+                    "id": "renderSurvey",
+                    "showDocs": true,
+                    "title": "renderSurvey",
+                    "examples": [
+                        {
+                            "id": "rendersurvey",
+                            "name": "Generated example for renderSurvey",
+                            "code": "// Generated example for renderSurvey\nposthog.renderSurvey();"
+                        }
+                    ],
+                    "releaseTag": "public",
+                    "params": [
+                        {
+                            "description": "",
+                            "isOptional": false,
+                            "type": "string",
+                            "name": "surveyId"
+                        },
+                        {
+                            "description": "",
+                            "isOptional": false,
+                            "type": "string",
+                            "name": "selector"
+                        }
+                    ],
+                    "returnType": {
+                        "id": "void",
+                        "name": "void"
+                    },
+                    "path": "lib/src/posthog-core.d.ts"
+                },
+                {
+                    "category": "",
+                    "description": "Resets all user data and starts a fresh session.",
+                    "details": "⚠️ **Warning**: Only call this when a user logs out. Calling at the wrong time can cause split sessions.\nThis clears: - Session ID and super properties - User identification (sets new random distinct_id) - Cached data and consent settings",
```

Comment:
Maybe this shouldn't be in a detail expand if it's important?

Reply:
Thanks, these pieces of feedback are very useful, but I'll address this in a separate PR to the posthog-js repo (since this is just generated from the actual code's comments). I'll link to these comments in that PR.

I'm more so focused on the code implementation in the website PR.